app/build.gradle

```diff
@@ -5,7 +5,7 @@ plugins {
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 23
+    compileSdkVersion 22
     buildToolsVersion "23.0.1"
```

Comment:
you need to drop this to 22.0.1 as well. See https://github.com/feedhenry-templates/saml-android-app/blob/master/app/build.gradle